pipelines/pipedrive/__init__.py

```diff
@@ -136,7 +148,8 @@ def deals_flow(deals_page, pipedrive_api_key=dlt.secrets.value):
         if data:
             yield from data
 
-if __name__=='__main__':
+
+if __name__ == '__main__':
```

Comment:
we really should not have `__main__` in `__init__`! we have `pipedrive_pipeline.py` for that

Reply:
thanks!